modules/graphstudio/pages/zoomChart-licenseExpiry-Upgrade.adoc

```diff
@@ -0,0 +1,116 @@
+= ZoomChart License Expiry and Deprecation Upgrade Instructions
+
+== Overview
+TigerGraph's license to use and distribute the ZoomCharts visualization library embedded within certain versions of GraphStudio will expire *December 31, 2023*.
+New versions of the TigerGraph Suite which *does not* use ZoomCharts has been developed and was released with TigerGraph 3.9.3.
+
+== Upgrade GraphStudio
+
+*We encourage you to perform an  upgrade as soon as possible to take advantage of the latest features and improvements in TigerGraph Suite.*
+Failure to upgrade before the expiry date may result in the display of a "License Expired" label in your graph visualizations.
+
+NOTE: No other features will be affected and this upgrade will not affect production use or result in any downtime for the upgrading system.
+
+
+If you have any questions or require assistance during the upgrade process, please don't hesitate to reach out to our support team at support@tigergraph.com. We are here to help you.
+
+Please follow the steps below based on your current database version.
+
+== Database Version 3.6.2, 3.6.3, or 3.6.4
+. Login to your TigerGraph instance's backend and change the current user to "tigergraph_system_user" (e.g., `sudo su tigergraph`).
+. Download the provided package to your database server and make sure under the TigerGraph installed path (e.g., `/home/tigergraph`).
```

Comment:
```suggestion
. Download the following package to your database server to the root folder of the TigerGraph installation (???) (e.g., `/home/tigergraph`).
```
This sentence has a typo, and I'm not 100% sure what the author meant to say.
Does the file need to be downloaded to /home/tigergraph?

Reply:
@samzhou2 can you clarify the instructions here in regards to @victorleeTG question?

Reply:
Yes, the file needs to be downloaded to /home/tigergraph folder

Reply:
Will update this with next commit.